packages/trader/src/App/Components/Elements/ToggleButton/__tests__/toggle-button-group.spec.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import ToggleButtonGroup from '../toggle-button-group.jsx';
+import ToggleButton from '../toggle-button.jsx';
+
+const MockToggleButtonGroup = ({ value, className, multiple, onChange }) => (
+    <ToggleButtonGroup value={value} className={className} onChange={onChange} multiple={multiple}>
+        <ToggleButton value='first-button'>first test button</ToggleButton>
+        {multiple && <ToggleButton value='second-button'>second test button</ToggleButton>}
+    </ToggleButtonGroup>
+);
+
+describe('ToggleButtonGroup', () => {
+    it('should have "toggle-button-group" class when "className" passed', () => {
+        render(<MockToggleButtonGroup className='test-class' />);
+        expect(screen.getByTestId('dt_toggle_button_group')).toHaveClass('toggle-button-group');
```

Comment:
in case of passing any className to it, it should render `toggle-button-group` ?

Reply:
> is there any reason that you changed the file extension to `js`?

The current files extension are 'jsx', I changed this test file to .ts when I was working on migrating enzyme task but because the main file is not typescript I had ts errors so I changed the extension back to .js

Reply:
> in case of passing any className to it, it should render `toggle-button-group` ?

Yep

Reply:
ok with file extension, thanks for clarification.
for the second one the behavior is odd!
so i pass `a` to a component and expect `b` to show as classname!
can you make it more clear?

Reply:
I made a mistake here, I'll fix this @ali-hosseini-deriv thank u for your revision :pray: